Classes/Common/Solr.php

```diff
@@ -276,7 +274,7 @@ public static function getSolrUrl($core = '')
             $host = $solrInfo['host'];
         }
         // Return entire request URL.
-        return $solrInfo['scheme'] . '://' . $host . ':' . $solrInfo['port'] . '/' . $solrInfo['path'] . '/' . $core;
+        return $solrInfo['scheme'] . '://' . $host . ':' . $solrInfo['port'] . '/solr/' . $core;
```

Comment:
See comment above. This should stay unchanged.

```suggestion
        return $solrInfo['scheme'] . '://' . $host . ':' . $solrInfo['port'] . '/' . $solrInfo['path'] . '/' . $core;
```